### ISSUE TITLE
size of `\Bigm|` is normal

### ISSUE THREAD

**Issue by hbghlyj** [ronkok/Temml#33]
Example
```
\Bigl \{a\in [0,n]\Bigm| \hbox {$a$ is coprime to $n$}\Bigr \} 
```
in [LaTeX](https://texlive.net/run?%5Cdocumentclass%7Barticle%7D%0A%5Cbegin%7Bdocument%7D%0A%5C%5B%0A%5CBigl%20%5C%7Ba%5Cin%20%5B0,n%5D%5CBigm%7C%20%5Chbox%20%7B%24a%24%20is%20coprime%20to%20%24n%24%7D%5CBigr%20%5C%7D%20%0A%5C%5D%0A%5Cend%7Bdocument%7D) the size of `\Bigm|` is bigger than normal
<img src="https://github.com/ronkok/Temml/assets/53823634/7715f49f-1acd-48b1-95b9-16b567f48b3d" height="80">
in [KaTeX](https://katex.org/) the size of `\Bigm|` is bigger than normal
<img src="https://github.com/ronkok/Temml/assets/53823634/e8c43049-2b40-4643-b707-906cb76cafa7" width="400">
But in Temml the size of `\Bigm|` is normal
![image](https://github.com/ronkok/Temml/assets/53823634/16f5bbe0-11fe-4f93-9940-3ffa4008f3c3)

**Comment by ronkok** [comment]
Yep, that's a bug. I'll fix it. A few days may go by before I get to it.

**Comment by ronkok** [comment]
Fixed by release 0.10.18.
